README.md

```diff
@@ -113,3 +113,30 @@ module.exports.GLYPHS = GLYPHS;
 var Icon = require('components/icon');
 <Icon glyph={Icon.GLYPHS.UNICORN}>
 ```
+
+Usage with React 0.14
+
+```js
+// icon.jsx
+const Icon = ({glyph, width, height, className}) => (
+  <svg className={className} width={width} height={height}>
+    <use xlinkHref={`${glyph}`} />
+  </svg>
+);
+
+Icon.defaultProps = {
+  width: 16,
+  height: 16,
+  className: 'icon'
+}
+
+export default Icon;
```

Comment:
Wait... what's the syntax for that. `export default const Icon` is a no-go. Also, am I breaking things by assigning to icon.defaultProps?